corehq/apps/hqadmin/views.py

```diff
@@ -1043,3 +1044,18 @@ def get_context_data(self, **kwargs):
         context = super(DimagisphereView, self).get_context_data(**kwargs)
         context['tvmode'] = 'tvmode' in self.request.GET
         return context
+
+
+def top_five_projects_by_country(request):
+    data = {}
+    if request.GET.get('country'):
```

Comment:
This is a nitpick, but `if 'country' in request.GET:` is a little more idiomatic I think